src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -1452,7 +1476,7 @@ ValidatorList::removePublisherList(
 std::size_t
 ValidatorList::count(ValidatorList::shared_lock const&) const
 {
-    return publisherLists_.size();
+    return publisherLists_.size() + (localPublisherList.list.size() > 0);
```

Comment:
No, this was not influenced by the unit tests.

The locally configured validator lists are mapped to a default-constructed Public Key in the `publisherLists_` map. Since we are removing such default-constructed public keys, I had to place that validator list in a separate variable.

Consequently, the validator list collection has been split into two mutually exclusive parts (the one from the local config file `localPublisherList` and the ones from other peers, which is stored in `publisherLists_` map)